src/components/tabell/brukerpanel-knapp.tsx

```diff
@@ -0,0 +1,27 @@
+import {MouseEvent} from 'react';
+import {Button} from '@navikt/ds-react';
+import {ChevronDownIcon, ChevronUpIcon} from '@navikt/aksel-icons';
+import './tabell.css';
+import '../../enhetsportefolje/brukerliste.css';
+
+interface BrukerpanelKnappProps {
+    apen: boolean;
+    onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
+}
+
+export const BrukerpanelKnapp = ({apen, onClick}: BrukerpanelKnappProps) => (
+    <Button
+        size="small"
+        className="brukerliste__brukerpanelknapp"
+        variant="tertiary"
+        onClick={onClick}
+        icon={
+            apen ? (
+                <ChevronUpIcon title="Vis mindre" className="collapse-testid" />
```

Comment:
Fiks at testid brukar test-id i staden for className (om mogleg)

Reply:
Det viser seg at berre arbeidslistetesten brukte desse testid'ane, så då kan vi likesågreit slette dei.